### ISSUE TITLE
about depth

### ISSUE THREAD

**Issue by MELANCHOLY828** [zhengzhang01/Pixel-GS#7]
Hello, I have tried to optimize the method provided in the paper for my application. However, I found that the scaling_factor is either 1 or 0, indicating that the depth is either very large or zero. I would like to ask if this is normal, as it seems that this removal of floaters has no effect.

**Comment by zhengzhang01** [comment]
Yes, this is normal. For a given view, the scaling_factor value of the Gaussian outside the view frustum is 0. Inside the view frustum, points farther from the camera do not scale the gradient, and the scaling_factor value is 1. As shown in the diagram, the red area represents the visible region, and only a small portion of the blue region undergoes gradient scaling.
![2](https://github.com/zhengzhang01/Pixel-GS/assets/133135464/ff26e5d1-0bd2-4c59-8c4e-c3d24b015064)

Yes, this is normal. For a given view, the scaling_factor value of the Gaussian outside the view frustum is 0. Inside the view frustum, points farther from the camera do not scale the gradient, and the scaling_factor value is 1. As shown in the diagram, the red area represents the visible region, and only a small portion of the blue region undergoes gradient scaling.

If you still notice floaters, you can adjust the depth_threshold parameter. The intuition behind this method is to suppress the splitting of Gaussians near the camera by reducing their gradients, thereby suppressing the floaters.
